### DISCUSSION TITLE
Memory usage tracking

### DISCUSSION THREAD

**Discussion by atorstling** [Viladoman/CompileScore#50] [Ideas]
Hello!

Love the tool, thank you very much for providing it. I was thinking; how about compiler memory usage tracking in addition to CPU usage? I find that useful to find things like inefficient template instantiation. It's usually correlated to CPU usage, but not always in a 1-1 manner.

**Comment by Viladoman** [comment]
Interesting idea! I don't think the compilers currently provide this information, but I guess some approximated data could be achieved by sampling the app mem usage at the OS level.
